Makefile

```diff
@@ -41,3 +41,7 @@ down: default
 .PHONY: build
 build: default
 	docker compose build
+
+.PHONY: clean-build
+clean-build: default
+	docker compose build --no-cache
```

Comment:
Does this need the `--profile="*"` that I needed when running locally, or was that just a marc thing?

Reply:
no, it gets that from the `default` bit on the line above